src/NServiceBus.Core/Licensing/GenerateReleaseDate.targets

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="GenerateReleaseDate" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+
+    <ParameterGroup>
+      <IntermediateOutputPath Required="true" />
+      <OriginalCommitDateAndTime />
```

Comment:
My initial naming decisions were around aligning with the already existing [ReleaseDateAttribute](https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/ReleaseDateAttribute.cs), so that's where "original" came from.

I thought about changing the naming, but to make changes to `ReleaseDateAttribute`, I'd also need to update `ReleaseDateReader`, but that comes from the licensing package, so I didn't want do that yet.

I suppose the code here could use some better names, and we can align everything else later.